### DISCUSSION TITLE
Would be good for those shorts prompt located in the page have abilities to be translated according to the language selected?

### DISCUSSION THREAD

**Discussion by peuportier** [open-webui/open-webui#2136] [Ideas]
It would be fantastic if we could enhance the settings interface by adding an option to switch languages directly from the menu, and similarly update the shortcut commands to reflect this new functionality seamlessly. Additionally, I'd love it if we could implement a step-by-step wizard guide for creating various content types easily. Thank you for considering these improvements.

**Comment by justinh-rahb** [comment]
Default prompts are kept in `/app/backend/data/config.json`, you could copy this out of the container and then bind mount it from the host so you can edit them:

*(this is based on our default `docker run` command, modify as necesary for your own case)*
```bash
docker cp open-webui:/app/backend/data/config.json .
docker run -d \
  -p 3000:8080 \
  --add-host=host.docker.internal:host-gateway \
  -v ./config.json:/app/backend/data/config.json \
  -v open-webui:/app/backend/data \
  --name open-webui \
  --restart always \
  ghcr.io/open-webui/open-webui:main
```

**Comment by peuportier** [comment]
Ok great, yes I check then, I can add for different local here. Thanks you. I will translate in my language and share with community.
Really helpful 
Thanks, Have a nice day
